src/renderer/state.ts

```diff
@@ -26,7 +27,7 @@ import {
   createMosaicArrangement,
   getVisibleMosaics,
 } from '../utils/editors-mosaic-arrangement';
-import { getName } from '../utils/get-title';
+import { getName } from '../utils/get-name';
```

Comment:
Yes I think we still need something like `state.name` because it's used by Runner to store data while running.

And it needs to be a property on the state object like this,  rather than computed, since it can be a semirandom string generated by namor in utils/get-name.ts. I'm still getting up to speed on mobx and am not sure if it needs to be an @action though.

...That said, we use it in `Runner::deleteUserData()` to 'Delet[e] data dir` but if this is the only place it's used then I don't see how the dir is getting created or what data is put in it :confused:

Reply:
followup: ah, it's in utils/get-package.ts` getPackageJson().

Reply:
remove the `action` key @ f7d4cc559984589b706304019207e939179d410c